jekyll/_cci2/credits.md

```diff
@@ -35,15 +35,70 @@ To set up your desired plan, go to `Settings > Plan Overview` in the CircleCI we
+### Expansive list of available resource classes
+{: #expansive-list-of-available-resource-classes }
+
+When using the free plan, you have the widest array of resource classes on Docker, Linux, Windows and macOS (coming soon) available to use. With this flexibility in choosing resource classes, you can be sure to choose the right compute resources for your job.
+
+For more information about these resources, please refer to the the [Executors and Images](https://circleci.com/docs/2.0/executor-intro/) page.
+
+### Concurrency and test splitting
+{: #concurrency-and-test-splitting }
+
+The ability to run multiple tests at the same time (concurrently) is a very powerful feature that allows you to decrease your build times and shorten feedback cycles by running jobs concurrently with workflows. Because one of the most important contributors to developer efficiency is being able to retrieve test results quickly, CircleCI can intelligently split tests to get you your test results faster using test splitting.
+
+For more information about how to utilize test splitting, refer to the [Running Tests In Parallel](https://circleci.com/docs/2.0/parallelism-faster-jobs/#using-test-splitting-with-python-django-tests) page.
+
+### Self hosted runners
+{: #self-hosted-runners }
+
+CircleCI runners allow you to use your own infrastructure for running jobs, providing more granular control of your own environment and flexibility in building and testing on a wide variety of architectures.
+
+For more information about using CircleCI runners, please see the [CircleCI Runner Overview](https://circleci.com/docs/2.0/runner-overview/) page.
+
+### Flaky test detection
+{: #flaky-test-detection }
+
+Flaky test detection is an important tool that helps you determine why a test may pass one time, and then fail the next time. Because the average failed workflow wastes approximately 30 minutes, having the ability to identify flaky tests can save you significant amounts of time in your builds and workflows.
+
+For more detailed information about flaky tests, refer to the [Test Insights](https://circleci.com/docs/2.0/insights-tests/#flaky-tests) page.
+
+### Unlimited private orbs
+{: #unlimited-private-orbs }
+
+CircleCI orbs are shareable configuration packages that enable developers to create private, standardardized configurations that can be used across an organization by different teams.
+
+For more information on private orbs, refer to the [Orbs Introduction](https://circleci.com/docs/2.0/orb-intro/#private-orbs-vs-public-orbs) page.
+
+### Docker layer caching
+{: #docker-layer-caching }
+
+Reducing the time it takes to build a Docker images is an important consideration for developers who consistently use these images in their workflows.  Docker Layer Caching (DLC) is a great feature to use if building Docker images is a regular part of your CI/CD process. DLC will save image layers created within your jobs, rather than impact the actual container used to run your job.
```

Comment:
```suggestion
Reducing the time it takes to build a Docker image is an important consideration for developers who consistently use these images in their workflows. Consider using Docker Layer Caching (DLC) if building Docker images is a regular part of your CI/CD process. DLC saves image layers created within your jobs, rather than impact the actual container used to run your job.
```